js/package.json

```
@@ -1,18 +1,19 @@
 {
-  "name": "@fof/split",
-  "version": "0.0.0",
-  "dependencies": {
-    "flarum-webpack-config": "0.1.0-beta.10",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
-  },
-  "scripts": {
-    "build": "webpack --mode production",
-    "dev": "webpack --mode development --watch",
-    "lint": "prettier --single-quote --trailing-comma es5 --print-width 150 --tab-width 4 --write src"
-  },
-  "devDependencies": {
-    "flarum": "0.1.0-beta.16",
-    "prettier": "^2.3.0"
-  }
+    "name": "@fof/split",
+    "version": "1.0.0",
+    "dependencies": {
+        "flarum-webpack-config": "1.0.0",
```

```suggestion
        "flarum-webpack-config": "^1.0.0",
```
Let's not constrain that hard..